lib/Conversion/SCFToCalyx/SCFToCalyx.cpp

```diff
@@ -172,6 +186,7 @@ class IfLoweringStateInterface {
   }
 
 private:
+  DenseMap<Operation *, calyx::RegisterOp> condReg;
```

Comment:
I know this hasn't been the norm, but it would sure be nice to describe why this data structure exists for future readers :-)